pywbem/cim_operations.py

```diff
@@ -5248,9 +5285,11 @@ def GetClass(self, ClassName, namespace=None, LocalOnly=None,
               causes the server-implemented default to be used. :term:`DSP0200`
               defines that the server-implemented default is `False`.
 
-          PropertyList (:term:`py:iterable` of :term:`string`):
-            An iterable specifying the names of the properties to be included
-            in the returned class (case independent).
+          PropertyList (:term:`py:iterable` of :term:`string` or :term:`string`):
+            An iterable specifying the names of the properties (or a  string
+            that defines a single property) to be included in the returned
+            instances (case independent).
```

Comment:
This paragraph should not talk about "properties to be included in the returned instances" but instead about "properties to be included in the returned class". See the original text.